docs/developer/api-reference/enums/webhook-event-type-async-enum.mdx

```diff
@@ -433,7 +433,7 @@ A shipping zone is deleted.
 
 #### [`STAFF_CREATED`](#)
 
-A new staff user is created.
+A staff user is deleted
```

Comment:
> A staff user is deleted

The event is called `STAFF_CREATED` but the content is "A staff user is deleted". Is that right?

Reply:
Hhm... Thanks for catching this. I will check this.